lmdeploy/pytorch/session.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from transformers.generation.utils import ModelOutput
+
+
+class BasicSessionManager:
+    """Basic session manager without history."""
+
+    def prepend_history(self, input_ids):
+        return input_ids
+
+    def add_to_history(self, output):
+        pass
+
+
+class BasicSessionManagerWithHistory:
+    """Basic session manager with chat history."""
+    bs = 1
+
+    def __init__(self,
+                 max_history=2048,
```

Comment:
better to use `session_len` for convention

Reply:
fixed